src/components/Loading/LoadingErrorIndicator.js

```diff
@@ -25,7 +29,7 @@ LoadingErrorIndicator.propTypes = {
   error: PropTypes.oneOfType([
     PropTypes.shape({ message: PropTypes.string }),
     PropTypes.bool,
```

Comment:
You can get rid of this, if you fix this `setError` statement - https://github.com/GetStream/stream-chat-react/blob/master/src/components/ChannelList/hooks/usePaginatedChannels.js#L56 This is the only place where we set error!!

Reply:
let's do this after, too annoying to refactor right now @vishalnarkhede